app/src/app/classes/authn/user-account.ts

```diff
@@ -116,11 +115,11 @@ export class UserAccount extends Serializable {
      * @returns {Observable} of the post
      */
      public save(restAPIService: RestApiConnectorService): Observable<UserAccount> {
-        let postObservable = restAPIService.postUserAccount(this);
-        let subscription = postObservable.subscribe({
+        const putObservable = restAPIService.putUserAccount(this);
+        const subscription = putObservable.subscribe({
             next: (result) => { this.deserialize(result.serialize()); },
             complete: () => { subscription.unsubscribe(); }
         });
```

Comment:
When changing a user's role on the user-accounts page, this code throws an error: `TypeError: result.serialize is not a function`

Reply:
Fixed!!! It was a tricky thing with the syntax--it had to both be `this.deserialize()` and `this.serialize()` and not `result.serialize()`, even though result is a User Account object

Reply:
Fixed!!! It was a tricky thing with the syntax--it had to both be `this.deserialize()` and `this.serialize()` and not `result.serialize()`, even though result is a User Account object

Reply:
Good catch! I wonder why the user-accounts class has this issue, but other stix objects don't. Glad you were able to find a fix for it though!